src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/IntegrationTestsExtension.java

```diff
@@ -16,6 +16,7 @@
 import java.util.Objects;
 
 import static org.springframework.jms.support.destination.JmsDestinationAccessor.RECEIVE_TIMEOUT_NO_WAIT;
+import static uk.nhs.digital.nhsconnect.lab.results.IntegrationBaseTest.DLQ_PREFIX;
```

Comment:
It seems that this import is not used

Reply:
```suggestion
```

Reply:
Ah, this should be used actually and the declared constant (duplicate) should be removed. I've fixed this now